server/swagger.json

```diff
@@ -3999,6 +3999,13 @@
 									"$ref": "#/definitions/DashboardQuery"
 								}
 							},
+              "axes": {
+                "description": "The viewport for a cell's visualizations",
+                "type": "object",
+                "additionalProperties": {
```

Comment:
I think I projected another meaning onto your comment when I read this yesterday. Will changing the documentation only be sufficient? I'd still like to keep the underlying protobufs as a `map<string,Axis>` because it's more flexible in the face of an uncertain future.